### ISSUE TITLE
Independent control over the color value of ellipses (GNA #8937)

### ISSUE THREAD

**Issue by wesnoth-bugs** [wesnoth/wesnoth#1105]
**[Original submission](https://gna.org/bugs/index.php?8937)** by eleazar on 2007-04-15

Currently the ellipses of many team colors blend into several kinds of terrain. (note the cavalier in the water)  Currently this ellipses are tied to the "middle" color of a unit's TColoring (i.e. the first one in the list.)  This does not allow the ellipse color to be adjusted independently of the unit's coloring, which leads to ineffectually colored ellipses, especially with "extreme" colors like yellow, black, and white where the "key" identifiable color is one of the extremes, not the middle of the TC continuum.

So, in short, please make the color of the ellipse controlled by a *5th* color value in the color_range definition.

(Reproduced on ALL)
Release: SVN
Priority: 3 - Low
Severity: 2 - Minor

**Comment by wesnoth-bugs** [comment]
**Modified on 2007-04-15**

**eleazar added attachment:** [ellipses.jpg](http://gna.org/bugs/download.php?file_id=2230)

**Comment by Pentarctagon** [comment]
Closing since, while it's not quite the same, the unit ellipses are now fairly configurable.  Also worth noting that the [color_range] definitions don't seem to have a 5th color value anymore.

**Comment by CelticMinstrel** [comment]
I think you misunderstood – this was proposing the _addition_ of a 5th colour. I imagine the issue described here is still present, but I don't necessarily agree with the proposed solution.
